transport/owr_transport_agent.c

```diff
@@ -1299,6 +1344,10 @@ static void link_rtpbin_to_send_output_bin(OwrTransportAgent *transport_agent, g
 
         if (dtls_srtp_bin_rtcp) {
             /* RTCP standalone */
+            /* FIXME: This might not be ready yet. In the worst case it might
+             * still be in NULL state, causing GST_FLOW_FLUSHING. Or it just
+             * didn't finish the handshake yet
+             */
```

Comment:
See below, this is the same problem. I'm not 100% sure anymore why it could be FLUSHING though. When do we change the state of the dtls_srtp_bin_rtcp and related elements? Maybe we should make sure to change their state here if it didn't happen before?